styles.css

```diff
@@ -187,13 +187,13 @@ main {
 .titulo {
     padding:20px;
     font-size:34px;
-    letter-spacing: 2px;
+    letter-spacing: 3px;
+    color:#36b37f;
```

Comment:
title color is not appropriate, check the [figma template](https://www.figma.com/design/fwIJIDv50WMoLpEEjc7Yqn/Portfolio-Inspo) for reference